koopovereenkomst-v3-simple/pages/brk.tsx

```diff
@@ -0,0 +1,55 @@
+import { useSession } from "@inrupt/solid-ui-react";
+import { Button } from "@mui/material";
+import { Box } from "@mui/system";
+import Head from "next/head";
+import { useEffect, useState } from "react";
+import ConnectSolid from "../src/ConnectSolid";
+import Layout from "../src/Layout";
+import VC, { VCType } from "../src/VC";
+
+export default function Brk() {
+
+    const { session } = useSession();
+    const [isLoggedOn, setIsLoggedOn] = useState(false);
+
+    const refreshState = () => {
+        setIsLoggedOn(session.info.isLoggedIn);
+        console.log("Refresh state. isLoggedOn: ", isLoggedOn);
+    }
+
+    useEffect(() => {
+        refreshState();
+        console.log("useEffect. isLoggedOn: ", isLoggedOn);
+    }, [])
```

Comment:
React Hook useEffect has missing dependencies: 'isLoggedOn' and 'refreshState'. Either include them or remove the dependency array